web-app/js/source/topicDialog.js

```diff
@@ -532,9 +532,55 @@ function deleteResource (resourceIDs) {
 $(
 	function () {
 		'use strict';
+		var numberOfCriticalTopics = 0;
+		var numberOfVeryImportantTopics = 0;
+		var numberOfGoodToKnowTopics = 0;
 		var savedData = true;
 
-		// Attach event listeners
+		$('input[id^="topicPrioritySaved"]').each(function () {
+			if ($(this).val() === 'Critical') {
+				numberOfCriticalTopics += 1;
+			}
+			if ($(this).val() === 'Very Important') {
+				numberOfVeryImportantTopics += 1;
+			}
+			if ($(this).val() === 'Good to Know') {
+				numberOfGoodToKnowTopics += 1;
+			}
+		});
+		$('#chart').chart({
+			data: new Array(numberOfCriticalTopics, numberOfGoodToKnowTopics, numberOfVeryImportantTopics),
```

Comment:
In JS the `Array` constructor takes in a single parameter `size`.
To initialize items in an `Array` use this instead
``` javascript
data: [numberOfCriticalTopics, numberOfGoodToKnowTopics, numberOfVeryImportantTopics]
```